.github/workflows/release.yml

```diff
@@ -27,12 +29,12 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Get the version
-        id: get_version
-        run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
-      - name: Create release branch
-        uses: peterjgrainger/action-create-branch@v2.0.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          branch: release/${{ steps.get_version.outputs.VERSION }}
+      - name: Get the new version
+        id: new_version
+        run: echo ::set-output name=VERSION::$(git tag --list "v*" --sort=refname | tail -n 1)
+      - name: Get the old version
+        id: old_version
+        run: echo ::set-output name=VERSION::$(git tag --list "v*" --sort=refname | tail -n 2 | head -n 1)
```

Comment:
@AnaMariaMoga - what you want here is actually `--sort=version:refname` . `refname` only sorts lexicographically  . e.g if you tagged in order v1.2.3, v2.3.4, v12.3.4, v23.3.4 - refname will give you 
```
$ git tag --list "v*" --sort=refname
v1.2.3
v12.3.4
v2.3.4
v23.3.4
```

If you run it with `--sort=version:refname` you'll get
```
v1.2.3
v2.3.4
v12.3.4
v23.3.4
```

Reply:
updated